knex/migrations/20200527204545_switch.js

```diff
@@ -0,0 +1,11 @@
+exports.up = function (knex) {
+    return knex.schema.table("writer_educations", (tbl) => {
+        tbl.renameColumn("writer_profile_id", "writer_id");
+    });
+};
+
+exports.down = function (knex) {
+    return knex.schema.table("writer_educations", (tbl) => {
+        tbl.renameColumn("writer_id", "writer_profile_id");
+    });
+};
```

Comment:
This migration would be renaming a foreign key that currently refers to the writer_profile_id. I think we're looking to make it refer to the user_id in the database. Would it be better to drop the "writer_profile_id" column and make a new one that relates to the user Id as a foreign key?